Twig/Extension/FormFlowExtension.php

```diff
@@ -36,8 +41,14 @@ public function getFilters() {
 	 * {@inheritDoc}
 	 */
 	public function getFunctions() {
+		if (version_compare(\Twig_Environment::VERSION, '1.12', '<')) {
```

Comment:
I won't drop support for Symfony 2.1 and 2.2 within the current 2.1 branch. And I also don't plan to create a 2.2 branch. So this has to wait for 3.0.